### ISSUE TITLE
Couple simple improvements.

### ISSUE THREAD

**Issue by Anonym0u8** [Ross-Patterson/Portals-Desktop-Organization#185]
**Is your feature request related to a problem? Please describe.**
No.

**Describe the solution you'd like**
***** Adjustable size for header bar. 
***    Adjustable orientation for header bar/Window.
****  Option for hover expand to open upwards. (Or sideways depending on oirentation/snap location)
***    Ability to snap to top, bottom and sides of display.

**Describe alternatives you've considered**

**Additional context**
Thanks for the hard work so far.

**Comment by Ross-Patterson** [comment]
*** Adjustable orientation for header bar/Window.
This is already available in the settings.

**** Option for hover expand to open upwards. (Or sideways depending on oirentation/snap location)
This is already available in the settings.

*** Ability to snap to top, bottom and sides of display.
I will add this to the backlog.

***** Proper text centering in header bar (with centering turned on) when size is adjusted, or icons (search, menu, etc) are turned on.
Can you explain what you mean why "Proper Centering"?

** Proper centering/alignment of window contents when size is adjusted, instead of defaulting to left and filling space on right.
Should be possible - will add it to the backlog to be investigated.

**Comment by Anonym0u8** [comment]
Oh, I guess I actually misunderstood the function of header bar placement. That indeed does change the first two things.

As far as the fourth thing - 
If you have any of the icons turned on, such as search, menu, drop down, etc - The text for the header bar will offset based on the amount of icons showing instead of centering with consideration of the icons size and position. So if you turn on all of the icons with a default position of the right side (and none on left) - the previously centered text is now far left. Maybe even just a manually adjustable header text position would be a fix or allowing user assignment of icon position. 
Maybe this was to prevent clipping? But if you make the header too small with icons then it will just replace the header text with "..." to prioritize keeping the icon, so I'm not sure.

Awesome program, finally set it all up - thanks again.

**Comment by Ross-Patterson** [comment]
Yeah I did it that way to prevent it being clipped too easily when the width of a Portal gets smaller.
